app/uk/gov/hmrc/pillar2stubs/models/UKTRSubmission.scala

```diff
@@ -26,6 +26,9 @@ sealed trait UKTRSubmission {
   val obligationMTT:        Boolean
   val electionUKGAAP:       Boolean
   val liabilities:          Liability
+
+  def accountingPeriodValid: Boolean =
+    accountingPeriodFrom.isBefore(accountingPeriodTo)
```

Comment:
shouldn't this be implemented in ET stub too?

Reply:
this should be a validation rule anyway, so should be implemented eventually